src/components/badge/badge.spec.tsx

```diff
@@ -110,5 +110,33 @@ describe("Badge", () => {
         wrapper.find(StyledBadge)
       );
     });
+
+    it("should render badge with default style when no customColor is specified", () => {
```

Comment:
```suggestion
    it("should render badge with default style when no color is specified", () => {
```

**nitpick** (non-blocking): I don't think we should refer to `customColor` any more now the prop has been renamed